### PR TITLE
BUG: stop fortran errors from crashing the interpreter

### PR DESCRIPTION
**Pull request by ghost**
This does not fix [github, it doesn't fix that issue] gh-8217, but it stops the interpreter crash in a generic way. Presumably this is okay because we've marked incorrect results as a defect, but not high priority for many cases.

Note that even though the functions don't raise an exception (which could presumably be implemented in the future), error messages are still printed.

### CODE REVIEW

**Comment by xoviat** [comment]
cc @person142 

Is this an appropriate fix?

File: scipy/integrate/odepack/xerrwv.f [hunk 1]
```diff
@@ -108,7 +108,7 @@ subroutine xerrwv (msg, nmes, nerr, level, ni, i1, i2, nr, r1, r2)
       if (nr .eq. 2) write (lun, 50) r1,r2
  50   format(6x,'in above,  r1 =',d21.13,3x,'r2 =',d21.13)
 c abort the run if level = 2. ------------------------------------------
```
Comment (rkern) [inline scipy/integrate/odepack/xerrwv.f]: Correct the comment. Probably remove the `stop` line as well.

**Comment by xoviat** [comment]
cc @rkern

**Comment by sturlamolden** [comment]
Note that LAPACK ```xerbla``` will crash the interpreter as well. The only "safe" way to call Fortran is to fork and run the Fortran code in a subprocess.

**Pull request closed without merging**